src/database/queries/changemakers/updateKeycloakOrganizationId.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE changemakers
```

Comment:
Would you be OK with sticking with our current (albiet little used) pattern of universal update queries with optional parameters?   You can see and example in `bulkUploadTasks/updateById.sql`.

This would just mean renaming the query to `updateById` -- I don't think you even need to use the COALESCE stuff that the bulkUploadTask query uses since there's only one field that can be updated here.

(The thinking is that we don't want to have to make an entire new query for every possible combination of fields that get updated for a given entity type.)